install_config/install/advanced_install.adoc

```diff
@@ -842,15 +844,66 @@ openshift_cockpit_deployer_prefix='registry.example.com/openshift3/'
 openshift_cockpit_deployer_version='3.9.3'
 ----
 
+
+[[advanced-install-glusterfs-persistent-storage]]
+=== Configuring {gluster} Persistent Storage
```

Comment:
Not sure if the {gluster} and {gluster-native} attributes work in the openshift-docs repo. @vikram-redhat Can you answer this question?

Reply:
They should work since I added the file `_snippets/glusterfs.adoc` and then include it by `include::_snippets/glusterfs.adoc[]` in every file that will use those variables.